answer_unit.php

```diff
@@ -381,8 +412,8 @@ private function parse_rules(&$mapping, &$dim_id_count, $rules_string) {
                     throw new Exception('Syntax error of SI prefix');
                 } else if (count($e) == 2) {
                     $unit_name = trim($e[0]);
-                    if (preg_match('/['.self::$unit_exclude_symbols.']+/', $unit_name)) {
-                        throw new Exception('"'.$unit_name.'" unit contains unaccepted character.');
+                    if (preg_match('/['.self::$rule_exclude_symbols.']+/', $unit_name)) {
+                        throw new Exception('"'.$unit_name.'" rule contains unaccepted character.');
```

Comment:
The string should be in `lang/en/qtype_formulas.php` and imported with get_string. (I know it wasn't done like this before, but if we're at it...)